### ISSUE TITLE
Why `ReverseRuleConfigBackend` but no `ForwardRuleConfigBackend`?

### ISSUE THREAD

**Issue by gdalle** [JuliaDiff/AbstractDifferentiation.jl#80]
If I understand correctly, AbstractDifferentiation is compatible with ChainRules backends that work in reverse mode, but not in forward mode?

**Comment by gdalle** [comment]
Yeah I did have Diffractor in mind, but also a potential translator from ChainRules to EnzymeRules. I don't think the PR would be hard, I can draft it if there is interest

**Comment by ChrisRackauckas** [comment]
ForwardDiff2 was essentially replaced by TaylorDiff.jl which will be using ChainRules for its vector rules.

**Comment by oxinabox** [comment]
It would be good to add this

**Comment by gdalle** [comment]
> It would be good to add this

Could this have something to do with a certain v0.2.0 release that just came out :innocent: ?

**Comment by gdalle** [comment]
> It would be good to add this

I can give it a shot but IIUC we can't test it on 1.9 since we need Diffractor?

**Comment by gdalle** [comment]
Also, the Diffractor backend already exists without being a special case of the (yet-to-be-written) `ForwardRuleConfigBackend`, is that an issue?
